openapi_schema_validator/shortcuts.py

```diff
@@ -16,8 +13,24 @@ def validate(
     *args: Any,
     **kwargs: Any
 ) -> None:
+    """
+    Validate an instance against a given schema using the specified validator class
+
+    Args:
+        instance: The instance to validate.
+        schema: The schema to validate against
+        cls: The validator class to use (defaults to OAS31Validator)
+        *args: Additional positional arguments to pass to the validator
+        **kwargs: Additional keyword arguments to pass to the validator
+
+    Raises:
+        jsonschema.exceptions.ValidationError: If the instance is invalid according to the schema
+    """
     cls.check_schema(schema)
     validator = cls(schema, *args, **kwargs)
-    error = best_match(validator.evolve(schema=schema).iter_errors(instance))
-    if error is not None:
+    errors = list(validator.evolve(schema=schema).iter_errors(instance))
+
+    if errors:
+        error = best_match(errors)
+        error.message = f"Validation failed: {error.message}"
```

Comment:
Would be nice to have it tested

Reply:
I studied a little more on my work project.
The solution I proposed adds another line to the log, although it's already clear from the output what's going on.

Returned the logic as it was before my edits.